portfolio/src/main/webapp/index.html

```diff
@@ -15,187 +15,132 @@
 
   <body>
     <nav>
-        <div class="nav-wrapper pink lighten-3">
+      <div class="nav-wrapper pink lighten-3">
         <a href="#" class="brand-logo"><i class="material-icons">star</i>Helena's Portfolio</a>
         <ul id="nav-mobile" class="right hide-on-med-and-down">
-            <!-- TODO(helenatran): Make the nav bar functional -->
-            <li><a href="#">My Projects</a></li>
-            <li><a href="#">Photo Gallery</a></li>
-            <li><a href="#">STEP Experience</a></li>
+          <!-- TODO(helenatran): Make the nav bar functional -->
+          <li><a href="./myProjects.html">My Projects</a></li>
```

Comment:
There may not be time this week, but something to think about... you've copied this portion of the html onto every page, generally speaking duplicating code is not great, particularly for a larger project that you'd have to maintain for a while. Another way you can do this is put the navbar html in it's own file and use the javascript to inject it into each page.

Reply:
Thanks for the review @lorikeet-s. I have made the necessary changes. 

Regarding this comment, I was thinking about looking at that today as, (apart from an extra feature I would like to implement but I'm not sure I will have enough time), I think my portfolio is practically done! :D

Reply:
Yeah if you don't have time don't worry about it, but if you're looking for things to do this might be something good to look at.